modules/openshift-cluster-maximums-major-releases.adoc

```diff
@@ -65,3 +70,20 @@ Tested Cloud Platforms for {product-title} 4.x: Amazon Web Services, Microsoft A
 4. There are a number of control loops in the system that must iterate over all objects in a given namespace as a reaction to some changes in state. Having a large number of objects of a given type in a single namespace can make those loops expensive and slow down processing given state changes. The limit assumes that the system has enough CPU, memory, and disk to satisfy the application requirements.
 5. Each service port and each service back-end has a corresponding entry in iptables. The number of back-ends of a given service impact the size of the endpoints objects, which impacts the size of data that is being sent all over the system.
 --
+
+.Example scenario
+
+You want to deploy 5,000 projects with 3 pods each and 6,000 projects with 2 pods.
+
+----
+5000 namespaces x 3 pods = 15000 pods
+6000 namespaces x 2 pods = 12000 pods
+----
+
+In total, you have 27,000 pods.
+
+----
+27000 pods / 250 pods per node = 108 nodes
+----
+
+The limit is 2,000 per node, therefore a 200 node cluster would work. However, there are 11,000 namespaces when the limit is 10,000. etcd might have issues handling this many active projects.
```

Comment:
2000 here is a number of nodes per cluster (it looks I wrote this wrong in BZ).
maybe something like:
Node maximum for cluster is 2000, therefore a 200 node cluster would work. However, there are 11,000 namespaces when the maximum is 10,000....

Reply:
I updated per your comment, let me know if that looks ok?